exec/jdbc-all/pom.xml

```diff
@@ -511,7 +511,7 @@
                   This is likely due to you adding new dependencies to a java-exec and not updating the excludes in this module. This is important as it minimizes the size of the dependency of Drill application users.
 
                   </message>
-                  <maxsize>39000000</maxsize>
+                  <maxsize>39050000</maxsize>
```

Comment:
Do you mean the size increase is because of adding new source files under java-exec modules ?